examples/moon.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+<head>
+<script src="https://aframe.io/releases/0.3.0/aframe.min.js"></script>
+<script src="../index.js"></script>
+</head>
+<body>
+<h1>This goes here!</h1>
+<!-- set up the scene -->
+<a-scene embedded altspace='vertical-align: bottom;' style='width: 600px; height: 400px;'>
+ 
+    <!-- preload all our textures -->
+    <a-assets>
+        <img id='moon' crossorigin src='https://developer.altvr.com/wp-content/uploads/2016/10/moon.jpg'/>
+        <img id='stars' crossorigin src='https://developer.altvr.com/wp-content/uploads/2016/10/stars.png'/>
```

Comment:
Probably add these images to this repo, just so that there isn't a dependency on the dev site.